assets/sass/components/key-metrics/_googlesitekit-km-selection-panel.scss

```diff
@@ -53,16 +53,31 @@
 
 	.googlesitekit-km-selection-panel-metrics {
 		overflow: auto;
+
+		.googlesitekit-km-selection-panel-metrics__subheading {
+			color: $c-surfaces-on-surface-variant;
+			font-size: $fs-body-sm;
+			font-weight: $fw-medium;
+			line-height: $lh-body-sm;
+			margin: $grid-gap-desktop 0 0 $grid-gap-desktop;
```

Comment:
It looks like we're missing a bit of responsiveness in the selection panel, these margins should be `$grid-gap-phone` at mobile viewports. Maybe this would be better addressed in a followup issue, though.